src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTaggerProvider.cs

```diff
@@ -56,9 +57,7 @@ internal class InlineHintsTaggerProvider : IViewTaggerProvider
 
         public ITagger<T>? CreateTagger<T>(ITextView textView, ITextBuffer buffer) where T : ITag
         {
-            // Determining of the textView's buffer does not match the buffer in order to skip showing the hints for
-            // the interactive window
-            if (buffer != textView.TextBuffer)
+            if (textView.IsBufferInInteractiveWindow(buffer))
```

Comment:
mentioned this before, but this really isn't anything to do with the interactive window.  this really sounds like it's just "is this on topmost view buffer".

Reply:
https://github.com/dotnet/roslyn/pull/48501#discussion_r508028334